conans/test/functional/generators/assets/cmake_find_package_multi/bye/src/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+if(CONAN_LINK_RUNTIME)
```

Comment:
Isn't this exactly the code that is in ``macro(conan_set_vs_runtime)``? I understand this has the limitation of not behaving the same in command line as in ``build()`` with CMake helper, but isn't it possible to embed that somehow in the generated code in the find packages or other file (as in conanbuildinfo.cmake) and reuse it?